tests/testdata/test_valid_knowledge_skill.yaml

```diff
@@ -1,176 +1,198 @@
+      (137,000 [light-years](light-years "wikilink")) in diameter and about
+      12.9 billion years old. [Robert's Quartet](Robert's_Quartet "wikilink")
+      (composed of the irregular galaxy [NGC 87](NGC_87 "wikilink"), and three
+      spiral galaxies [NGC 88](NGC_88 "wikilink"), [NGC 89](NGC_89 "wikilink")
+      and [NGC 92](NGC_92 "wikilink")) is a group of four galaxies located
+      around 160 million light-years away which are in the process of colliding
+      and merging. They are within a circle of radius of 1.6 arcmin,
+      corresponding to about 75,000 light-years. Located in the galaxy ESO
+      243-49 is [HLX-1](HLX-1 "wikilink"), an
+      [intermediate-mass black hole](intermediate-mass_black_hole
+      "wikilink")—the first one of its kind identified. It is thought to be a
+      remnant of a dwarf galaxy that was absorbed in a
+      [collision](Interacting_galaxy "wikilink") with ESO 243-49. Before its
+      discovery, this class of black hole was only hypothesized.
     questions_and_answers:
-      - question: What is a tonsillolith?
+      - question: |
+          Is the Phoenix Constellation part of the Milky Way?
+        answer: |
+          The Phoenix constellation does not lie on the galactic plane of
+          the Milky Way, and there are no prominent star clusters.
+      - question: |
+          How many light years away is NGC 625?
         answer: |
-          A tonsillolith (tonsil stone) is material that accumulates on the
-          palatine tonsil, reaching the size of a peppercorn and having a white
-          or cream color. It contains calcium and has a strong unpleasant odor
-          due to hydrogen sulfide, methyl mercaptan, and other chemicals.
-      - question: What is the main substance found in a tonsillolith?
-        answer: The main substance found in a tonsillolith is mostly calcium.
-      - question: Why do tonsilloliths have a strong unpleasant odor?
+          NGC 625 is 24000 light years in diameter and is an outlying
+          member of the Sculptor Group.
+      - question: |
+          What is Robert's Quartet composed of?
         answer: |
-          Tonsilloliths have a strong unpleasant odor due to hydrogen sulfide,
-          methyl mercaptan, and other chemicals.
-
+          Robert's Quartet is composed of the irregular galaxy NGC 87,
+          and three spiral galaxies NGC 88, NGC 89 and NGC 92.
 document_outline: |
-  Overview of Human tonsils, describing their types, locations, structure,
-  function, and clinical significance, with a specific focus on their role in
-  the immune system and related health issues.
-
+  Information about the Phoenix Constellation including the
+  history, characteristics, and features of the stars in the constellation.
 document:
-  repo: https://github.com/luke-inglis/il-anatomy-knowledge
-  commit: cc7c6ca
+  repo: https://github.com/RedHatOfficial/rhelai-taxonomy-data
```

Comment:
Is it ok to refer to a RHEL AI taxonomy here? Do members of the InstructLab team control this repository sufficiently to tweak it as needed if we need to adjust what we're testing?

Reply:
We have control over this repository, as this is meant for test cases and we can also make changes as and when needed.

Reply:
Ok - this seems reasonable to use then.